README.md

```diff
@@ -70,6 +70,7 @@ sudo make install
  | vx_min                     | double | Min VX                                                                                                                                                                                                                                                                                                               |
  | wz_max                     | double | Max WZ                                                                                                                                                                                                                                                                                                               |
  | temperature                | double | Selectiveness of trajectories by their costs (The closer this value to 0, the "more" we take in considiration controls with less cost), 0 mean use control with best cost, huge value will lead to just taking mean of all trajectories without cost consideration                                                   |
+ | gamma                | double |  A trade-off between smoothness (high) and low energy (low). This is a complex parameter that likely won't need to be changed from the default of `0.1` which works well for a broad range of cases. See Section 3D-2 in "Information Theoretic Model Predictive Control: Theory and Applications to Autonomous Driving" for detailed information.       |
```

Comment:
Need to add all new parameters to readme + update defaults in example below (+ in code)

Reply:
and tune